api/v1/storagecluster_types.go

```diff
@@ -254,6 +254,7 @@ type ManageCephRBDMirror struct {
 type MgrSpec struct {
 	// EnableActivePassive can be set as true to deploy 2 ceph manager pods, one active and one standby
 	// Ceph will promote the standby mgr when the active mgr goes down due to any reason
+	// +kubebuilder:deprecatedversion:warning="This field has been deprecated and will be removed in future versions. ActivePassive Mgr is now default."
```

Comment:
```suggestion
	// +kubebuilder:deprecatedversion:warning="This field has been deprecated and will be removed in future versions. Both the active and passive mgrs are always enabled."
```

Reply:
Reworded a bit to make it more informative.